frontend/src/components/Modal.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import "./style/Modal.scss";
+
+const Modal = ({ isShowing, hide, user_name }) =>
```

Comment:
I'm almost sure you just reinventd the wheel (Modali, react-modal, etc...), but ok if that works